### PR TITLE
Config units

### PR DESCRIPTION
**Pull request by LukasNickel**
Allows to set units for the columns:

- pointing_az/zd
- source_az/zd
- delta

These can be set in the DISP config. Defaults are "deg" for all az/zd columns and "rad" for delta.

!! This will break the CTA analyses, because the explicit deg2rad(delta) is gone!!
Your will need to specify the unit as "deg" as is done in the `examples/config_source_cta.yaml`.

Further ideas:
- Do we need units for focal length and camera coordinates (mm/m)?
- Do we want to specify the unit of the Energy (GeV/TeV)? If so: Would this be a global option?
- Are there any other columns that we need to watch out for?

ToDo:
- [x] add units for the angles and transform values
- [x] add unit for the focal length and transform values
- [x] add unit for the energy and use it for plotting labels
- [x] add tests for the DISP results -> not needed for now
- [x] add tests for the plot scripts -> #129

### CODE REVIEW

**Comment by MaxNoe** [comment]
focal_length: yes
energy: yes, but only needed for the plots (axis labels)

of the top of my head I think that's all, let me check

File: aict_tools/preprocessing.py [hunk 1]
```diff
@@ -76,3 +77,36 @@ def calc_true_disp(source_x, source_y, cog_x, cog_y, delta, project_disp=False):
...
+    coordinate_units = (
+        (model_config.delta_column, model_config.delta_unit, 'rad'),
+        (model_config.source_az_column, model_config.source_az_unit, 'deg'),
+        (model_config.source_zd_column, model_config.source_zd_unit, 'deg'),
+        (model_config.pointing_az_column, model_config.pointing_az_unit, 'deg'),
+        (model_config.pointing_zd_column, model_config.pointing_zd_unit, 'deg'),
+    )
+    for column, unit, expected_unit in coordinate_units:
+        if unit != expected_unit:
+            converted_values = u.Quantity(df[column], unit).to(expected_unit).value
```
Comment (MaxNoe) [inline aict_tools/preprocessing.py:109]: `df[colum].to_numpy(), copy=False` and `.to_value(unit)`

File: aict_tools/preprocessing.py [hunk 2]
```diff
@@ -99,14 +99,18 @@ def sanitize_angle_units(df, model_config):
...
         (model_config.pointing_az_column, model_config.pointing_az_unit, 'deg'),
         (model_config.pointing_zd_column, model_config.pointing_zd_unit, 'deg'),
     )
     for column, unit, expected_unit in coordinate_units:
         if unit != expected_unit:
-            converted_values = u.Quantity(df[column], unit).to(expected_unit).value
+            converted_values = u.Quantity(
+                    df[column].to_numpy(),
+                    unit,
+                    copy=False).to_value(expected_unit)
```
Comment (MaxNoe) [inline aict_tools/preprocessing.py:113]: comma and linebreak after `False `

**Comment by MaxNoe** [comment]
I added tests for the plotting scripts here: https://github.com/fact-project/aict-tools/pull/129

**Comment by MaxNoe** [comment]
Testing disp results would be nice, but I also run the open_crab_sample_analysis before each release is made as an integration test. So at least the FACT chain is fully tested.

**Comment by LukasNickel** [comment]
I suppose the failing tests are related to the sklearn2pmml requirements, that you changed in #129 @MaxNoe ?

**Comment by MaxNoe** [comment]
Yes

**Comment by MaxNoe** [comment]
Actually, now with the master merged, also the plot_disp_performance was failing, fixed it.

**Pull request merged**